templates/inspec/doc_template.md.erb

```diff
@@ -33,7 +33,7 @@ Properties that can be accessed from the `<%= resource_underscored_name -%>` res
 
 <% if plural -%>
 See <%= "[#{resource_name(object, product_ns)}.md](#{resource_name(object, product_ns)}.md)" -%> for more detailed information
-<% object.all_user_properties.each do |prop| -%>
+<% object.all_user_properties.reject { |prop| plural && prop.exclude_plural }.each do |prop| -%>
```

Comment:
```suggestion
<% object.all_user_properties.reject(&:exclude_plural).each do |prop| -%>
```

Line 34 makes it look like plural will always be true.

Reply:
+1 good catch